ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -25,32 +27,64 @@ int main()
         std::cout << "Connecting to the camera" << std::endl;
         auto camera = zivid.connectCamera();
 
-        std::cout << "Configuring the camera settings" << std::endl;
-        camera << Zivid::Settings::Iris{ 21 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 20000 } }
-               << Zivid::Settings::Gain{ 1 } << Zivid::Settings::Brightness{ 1.0 };
+        std::cout << "Enter <a><Return> for 2D only, and just <Return> for 3D" << std::endl;
+        const auto command = getInput();
+        bool use2D = false;
+        if(command == "a" || command == "A")
+        {
+            use2D = true;
+        }
+
+        cv::Mat rgb;
+
+        if(use2D)
+        {
+            std::cout << "2D only" << std::endl;
```

Comment:
If you print the mode when using 2d, I suggest printing it when using 3d as well, for consistency.